### DISCUSSION TITLE
No docker image for mysql

### DISCUSSION THREAD

**Discussion by fonteneu** [dcm4che/dcm4chee-arc-light#4602] [Q&A] [answered]
https://hub.docker.com/u/dcm4che?page=1&search=mysql has no hits, only psql

I need a non-secured version for MySQL. Do I have to build this Docker image myself?

**Comment by vrindanayak** [comment, marked as answer]
Archive with MySQL backend is only provided as a release package as any other database - other than postgres - which can be retrieved from [Releases link](https://sourceforge.net/projects/dcm4che/files/dcm4chee-arc-light5/5.33.0/)

**Reply by fonteneu** [reply]
Thanks. Are there instruction to make this into a docker image like the postgres one?

**Reply by vrindanayak** [reply]
There are no explicit instructions, but you may build docker image on similar lines as done for postgres by adjusting the [Dockerfile](https://github.com/dcm4che-dockerfiles/dcm4chee-arc-psql/blob/master/Dockerfile) and the [Wildfly Configuration](https://github.com/dcm4che-dockerfiles/dcm4chee-arc-psql/blob/master/configuration/dcm4chee-arc.xml) file to include MySQL DB info instead of PostgreSQL
